examples/udf.java/README.md

```diff
@@ -20,21 +20,21 @@ Output:
 #### Install `zillabase`
 
 ```bash
-$ brew tap aklivity/tap
+brew tap aklivity/tap
 
-$ brew install zillabase
+brew install zillabase
 ```
 
 Note: To use a local build, you can define an alias as follows:
 
 ```bash
-$ alias zillabase="java -jar `pwd`/cli/target/cli-develop-SNAPSHOT.jar"
+alias zillabase="java -jar `pwd`/cli/target/cli-develop-SNAPSHOT.jar"
 ```
```

Comment:
I think we elected to remove this everywhere, maybe it was still around in the udf.java example.